packages/flutter/lib/src/widgets/platform_view.dart

```diff
@@ -580,3 +580,71 @@ class _UiKitPlatformView extends LeafRenderObjectWidget {
     renderObject.updateGestureRecognizers(gestureRecognizers);
   }
 }
+
+/// This widget provides the minimal required functionality for a platform view.
+///
+/// The render object associated with this widget is [PlatformViewRenderBox].
+///
+/// This widget exists as a helper for easier implementation of a new platform view.
+/// It is entirely up to the implementer to decide whether to use this helper.
```

Comment:
What is the minimal functionality that adds?  How do I , as someone who thinks they want to implement this, use this?  How do I decide if I should use this or something else?

Reply:
+1 the documentation should answer these questions.
I suggest we just explicitly describe what it takes care of.

I wouldn't call this "minimal functionality" as the functionality here alone is not enough to embed a platform view.